README.md

```diff
@@ -435,6 +355,347 @@ One App can be started via docker or if built from source by running `node lib/s
 
 <!-- TODO talk about prometheus and logging schema here -->
 
+## 🎛️ API
```

Comment:
I like this structure! For the server API we could add the env vars and that type of config. 👍

Reply:
Yes, this PR is kinda a stab at a foundation for a API doc styleguide.